logging-infrastructure/README.md

```diff
@@ -0,0 +1,62 @@
+# Logging infrastructure
+
+As a Giant Swarm engineer, I want to be able to access a history of logs for components managing our platform in order to both investigate ongoing operational issues and provide details for incident reports.
```

Comment:
We might want to share this infra with customers, so the use case might be for both GS and customers. While they are similar the amount of data and its respective sensitivity with the customer use case is higher.

Reply:
+1, with the whole of idea of being a developer platform, i think customers having access to logs is a requirement

Reply:
Sure, the idea behind adding it is that considerations around the customer can already influence the architecture decision we make here (some of the customer constraints are mentioned below anyway) and we ideally would avoid re-architecting. Especially, if we are thinking about using the same system for customers and ourselves in a multi-tenant model.